src/predicates.ts

```diff
@@ -479,8 +487,8 @@ export class ConfigOptionExists implements Predicate<any> {
 
   private configItemNames(section: ConfigSection) {
     return _.flatMap(section.items, item => {
-      const hasChildren = ["select_one", "select_many"].indexOf(item.type) === -1;
-      return hasChildren ? [item.name] : _.map(item.items!, i => i.name);
+      const hasChildren = ["select_one", "select_many"].indexOf(item.type) !== -1;
+      return hasChildren ? [item.name, ..._.map(item.items!, i => i.name)] : [item.name];
```

Comment:
the name of `hasChildren` was misleading, it was `false` for items that have children. Flipped the boolean, plus fixed a bug where the parent item name wasn't included in the list of valid item names.